### DISCUSSION TITLE
Sonoff Dual R3... KO.

### DISCUSSION THREAD

**Discussion by Max-Br1** [indomus/forum#1615] [Hardware & Firmware]
Da ieri tutti i Sonoff Dual R3 configurati come "COVER" non vengono più rilevati da HA, mentre tutti quelli configurati come doppi interruttori, vanno ancora senza problemi.
Hanno funzionato fino a ieri e quasi sicuramente fino a prima dell'ultimo aggiornamento di HA.
C'è qualche settaggio da fare o bisogna attendere un ulteriore aggiornamento che mi auguro giunga a breve?
Di tutta la domotica se ti vengono meno proprio le tapparelle.... :-(

**Comment by snakuzzo** [comment]
Se non dai ulteriori dettagli è impossibile darti una mano

**Comment by indomus** [comment]
> Da ieri tutti i Sonoff Dual R3 configurati come "COVER" non vengono più rilevati da HA, mentre tutti quelli configurati come doppi interruttori, vanno ancora senza problemi. Hanno funzionato fino a ieri e quasi sicuramente fino a prima dell'ultimo aggiornamento di HA. C'è qualche settaggio da fare o bisogna attendere un ulteriore aggiornamento che mi auguro giunga a breve? Di tutta la domotica se ti vengono meno proprio le tapparelle.... :-(

che firmware montano? come sono configurati? risultano connessi alle rete? se li comandi manualmente, funzionano?
bisogna spiegare meglio le cose, gente...

**Comment by Max-Br1** [comment]
Calmi, non vi arrabbiate... Sto imparando :-)
Scherzi a parte, ho solo cercato di capire se il problema fosse solo mio o magari generalizzato a seguito dell'aggiornamento di H.A.
I Dual R3 funzionano sia in manuale (tasto fisico) che in rete mediante l'App eWeLink e Alexa.
Se dico "Alexa, tapparella camera 50", la tapparella si muove e si porta al 50%.
Il firmware è quello originale di fabbrica sonoff v. 1.4.0.
Tutti i dispositivi Sonoff li ho integrati in massa in H.A. con la repository "Sonoff LAN" mediante HACS.
Su tutti gli altri dispositivi Sonoff Mini R2 nessun problema sia con l'App, sia con Alexa e sia in H.A.
Stessa cosa anche per gli altri Dual R3 configurati come "doppio switch" funzionano sia da App, Alexa e H.A.
Il problema della "sparizione" riguarda solo gli R3 configurati come "COVER" che uso per domotizzare le tapparelle e che fino a due giorni fa funzionavano anche in H.A.
Adesso sono "scomparsi" dalla lista dei DISPOSITIVI e me li ritrovo nella lista delle ENTITA', tra l'altro DISATTIVATE, ma anche se li attivo, nulla di fatto e tutto resta tale e quale.
Credo che i comandi funzionino come "Switch" (canale 1 e canale 2)... ma non avrebbe alcun senso usarli in quella maniera senza alcun controllo, e per questo motivo non ci ho nemmeno provato.
Nessun errore nei LOG.
questo è ciò che ottengo dall'elenco "Stumenti per Sviluppatori":

![Immagine_1](https://user-images.githubusercontent.com/101845499/161583421-a0df59f7-3052-4d9a-9551-4d67ba924fec.jpg)

![Immagine_2](https://user-images.githubusercontent.com/101845499/161583463-d86ee30e-bcff-4343-bfc5-7e405e2acbb0.jpg)

![Immagine_3](https://user-images.githubusercontent.com/101845499/161585127-95419eb8-5b40-489d-843b-8062ddd83d72.jpg)

![Immagine_4](https://user-images.githubusercontent.com/101845499/161585165-322d5c85-4607-4032-bf2f-8c425dd518f3.jpg)

Spero di aver fornito abbastanza dettagli da poterci capire qualcosa in più.
